### DISCUSSION TITLE
Task Connectivity

### DISCUSSION THREAD

**Discussion by Nidesh-Arumugam** [apache/airflow#20068] [Q&A] [answered]
How to perform Task connectivity between a series of tasks. For example if my  first task(t1) is a bash command[mkdir foldername], and in the next task(t2) if i use  another  bash command[ls] i need the folder name to be displayed instead of being blank? got any idea?

suggestion would be much appreciated :)

**Comment by potiuk** [comment, marked as answer]
https://airflow.apache.org/docs/apache-airflow/stable/concepts/xcoms.html

**Reply by Nidesh-Arumugam** [reply]
can xcoms be used with bash operators? instead of returning a value from a  python function and using them in bash commands, i need a connectivity between two bash tasks

**Reply by josh-fell** [reply]
Yes. From the operator's docstring:

>If BaseOperator.do_xcom_push is True, the last line written to stdout
    will also be pushed to an XCom when the bash command completes

The `do_xcom_push` arg defaults to True for all operators so as long as the last line printed to `stdout` contains what you wish to push as an `XCom` it should work.
